sigma/backends/panther/helpers/pantherflow_helper.py

```diff
@@ -0,0 +1,145 @@
+from typing import Any, Union
+
+from sigma.conditions import (
+    ConditionAND,
+    ConditionFieldEqualsValueExpression,
+    ConditionOR,
+    ConditionValueExpression,
+    ParentChainMixin,
+)
+from sigma.conversion.state import ConversionState
+from sigma.exceptions import SigmaFeatureNotSupportedByBackendError
+
+from sigma.backends.panther.helpers.base import BasePantherBackendHelper
+
+
+class PantherFlowHelper(BasePantherBackendHelper):
+    WILDCARD_SYMBOL = "*"
+
+    LOG_TYPES_MAP = {
+        "Crowdstrike.FDREvent": "crowdstrike_eventstreams",
```

Comment:
Should be crowdstrike_fdrevent

```suggestion
        "Crowdstrike.FDREvent": "crowdstrike_fdrevent",
```